install-global.sh

```
@@ -0,0 +1,167 @@
+                                                              
+EOF
+echo -e "${NC}"
+
+echo "🚀 Claude グローバル設定インストーラー"
+echo ""
+
+# バックアップ関数
+backup_if_exists() {
+    local file=$1
+    if [[ -f "$file" ]]; then
+        local backup="${file}.backup.$(date +%Y%m%d_%H%M%S)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)